src/cljs/org/broadinstitute/firecloud_ui/common/components.cljs

```diff
@@ -113,6 +114,24 @@
         [:div {} (:component (nth (:items props) (:active-tab-index @state)))]])}))
 
 
+(react/defc AnchoredDialog
```

Comment:
I have used this concept elsewhere, but I added an onClick handler to the window. This pattern (adding an element behind the box to catch the click) results in a much simpler and cleaner implementation. Really nice.